### ISSUE TITLE
Strategy Evaluation

### ISSUE THREAD

**Issue by Malnammi** [gitter-lab/active-learning-drug-discovery#3]
See strategy at #1.

Currently, strategy [evaluation ](https://github.com/gitter-lab/active-learning-drug-discovery/blob/master/active_learning_dd/utils/evaluation.py#L47) looks at 3 main metrics for each selected batch: 
1. **n_hits**: how many hits.
2. **norm_hits_ratio**: the ratio of the number of hits to the max number of hits possible for this batch. Max hits is a function of the remaining unlabeled instances and the `batch_size`; i.e. if we have 100 remaining unlabeled hits and `batch_size=96`, then `max_hits=min(96, 100)`. 
3. **n_cluster_hits**: how many unique clusters with actives. Note this does not necessarily look at new clusters found (clusters not in the training set), it merely computes the number of unique clusters with hits in the selected batch. 
4. **norm_cluster_hits_ratio**: similar to **norm_hits_ratio** but for cluster hits.
5. **novelty_n_hits**: This was a metric @agitter and I discussed in the past. The formula is 
**novel_n_hits** = w * **norm_hits_ratio** + (1-w) * **norm_cluster_hits_ratio**

These metrics can then be used to compare different strategies.
